src/main/kotlin/com/mdrobnak/lalrpop/psi/LpResolveType.kt

```diff
@@ -4,6 +4,11 @@ import com.intellij.psi.PsiElement
 
 data class NonterminalGenericArgument(val rustType: String, var name: String)
 
+data class LpTypeResolutionContext(val locationType: String = "usize", val errorType: String = "()", val tokenType: String = "&str") {
+    fun errorRecovery() = "::lalrpop_util::ErrorRecovery<$locationType, $tokenType, $errorType>"
+    fun parseError() = "::lalrpop_util::ParseError<$locationType, $tokenType, $errorType>"
```

Comment:
Can these be changed to properties? Then all of the context's values are accessed in a similar way.
```suggestion
    val errorRecovery = "::lalrpop_util::ErrorRecovery<$locationType, $tokenType, $errorType>"
    val parseError = "::lalrpop_util::ParseError<$locationType, $tokenType, $errorType>"
```